src/add-remove.js

```diff
@@ -93,42 +93,48 @@ function AddTask(description) {
   taskArray.push(new Task(description));
   localStorage.setItem('taskArray', JSON.stringify(taskArray));
   addInput.value = '';
-  DisplayTask();
+  DisplayTask(taskArray);
+}
+
+function EditTesting(indexToUpdate, taskEdit) {
+  taskArray[indexToUpdate - 1].description = taskEdit;
+  localStorage.setItem('taskArray', JSON.stringify(taskArray));
+  DisplayTask(taskArray);
 }
 
 function EditTask(indexToUpdate, descriptionToUpdate) {
   // eslint-disable-next-line
   const taskEdit = prompt(`Editing '${descriptionToUpdate}' task`, descriptionToUpdate);
   if (taskEdit != null) {
-    taskArray[indexToUpdate - 1].description = taskEdit;
-    localStorage.setItem('taskArray', JSON.stringify(taskArray));
-    DisplayTask();
+    EditTesting(indexToUpdate, taskEdit);
   }
 }
 
 function DeleteTask(indexToDelete) {
   taskArray = taskArray.filter((task) => task.index !== indexToDelete);
   indexToAssign -= 1;
   UpdateIndexes();
-  DisplayTask();
+  DisplayTask(taskArray);
 }
 
-function DeleteAllCompletedTasks() {
-  if (localStorage.getItem('taskArray')) {
-    taskArray = JSON.parse(localStorage.getItem('taskArray'));
-  }
+function DeleteAllCompletedTasks(taskArray) {
+  // if (localStorage.getItem('taskArray')) {
+  //   taskArray = JSON.parse(localStorage.getItem('taskArray'));
+  // }
```

Comment:
With the new update, you introduced a bug, this function seems not working anymore, try to test this on the UI.

kindly remove commented code.